server/pkg/accounts/accounts.go

```diff
@@ -3,14 +3,39 @@ package accounts
 import (
 	"fmt"
 	"os"
+	"errors"
 
 	"github.com/darwinfroese/hacksite/server/models"
 	"github.com/darwinfroese/hacksite/server/pkg/auth"
 	"github.com/darwinfroese/hacksite/server/pkg/database"
 )
 
+const (
```

Comment:
Hmm, this was/is in the auth package but doesn't look like it's used here. 

Did you have a reason for pulling this in?

Reply:
Nope! Was an early attempt at getting things to work.